package.json

```diff
@@ -32,10 +34,12 @@
     "@tsconfig/react-native": "3.0.5",
     "@types/node": "^22.13.0",
     "eslint": "^9.19.0",
+    "eslint-config-prettier": "^10.1.5",
     "globals": "^16.0.0",
+    "prettier": "3.5.3",
```

Comment:
Let's upgrade this to Prettier 3.6 (see [blog post](https://prettier.io/blog/2025/06/23/3.6.0)) for mad speed.